packages/components/src/components/NewModal/types.tsx

```diff
@@ -1,3 +1,7 @@
-import { UIVariant } from '../../config/types';
+import { UIVariant, UISize, UIHorizontalAlignment, UIVerticalAlignment } from '../../config/types';
 
 export type NewModalVariant = Extract<UIVariant, 'primary' | 'warning' | 'destructive'>;
+
+export type NewModalSize = Extract<UISize, 'large' | 'medium' | 'small'>;
```

Comment:
Btw this might be refactored in the future. Check this PR: https://github.com/trezor/trezor-suite/pull/13828